<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgDlgResourceManager</class>
 <widget class="QWidget" name="WdgDlgResourceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidget_1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbResourceType"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmbStorage"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmbTag"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chkShowDeleted">
         <property name="text">
          <string>Show deleted resources</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KisResourceItemListView" name="resourceItemView"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lneFilterText">
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblThumbnail">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>Preview</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="vertLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="KSqueezedTextLabel" name="lblId">
               <property name="textInteractionFlags">
                <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="KSqueezedTextLabel" name="lblMetadata">
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Filename:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tags:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lneName"/>
         </item>
         <item row="2" column="1">
          <widget class="KSqueezedTextLabel" name="lblLocation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KSqueezedTextLabel" name="lblFilename">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KisTagSelectionWidget" name="wdgResourcesTags" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteResource">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete Resources</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnImportResources">
           <property name="text">
            <string>Import Resources</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnOpenResourceFolder">
           <property name="text">
            <string>Open Resource Folder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnCreateBundle">
           <property name="text">
            <string>Create a Bundle</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnDeleteBackupFiles">
           <property name="text">
            <string>Delete Backup Files</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>ksqueezedtextlabel.h</header>
  </customwidget>
  <customwidget>
   <class>KisResourceItemListView</class>
   <extends>QListView</extends>
   <header location="global">KisResourceItemListView.h</header>
  </customwidget>
  <customwidget>
   <class>KisTagSelectionWidget</class>
   <extends>QWidget</extends>
   <header>KisTagSelectionWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbResourceType</tabstop>
  <tabstop>cmbStorage</tabstop>
  <tabstop>cmbTag</tabstop>
  <tabstop>chkShowDeleted</tabstop>
  <tabstop>resourceItemView</tabstop>
  <tabstop>lneFilterText</tabstop>
  <tabstop>lneName</tabstop>
  <tabstop>btnDeleteResource</tabstop>
  <tabstop>btnImportResources</tabstop>
  <tabstop>btnOpenResourceFolder</tabstop>
  <tabstop>btnCreateBundle</tabstop>
  <tabstop>btnDeleteBackupFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
